<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>José Manuel Díez Botella</author>
 <class>preferencias</class>
 <widget class="QDialog" name="preferencias">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferencias</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Parámetros</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Ventana principal</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="automaticoscheckBox">
              <property name="text">
               <string>&amp;Asientos automáticos</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="estadoscheckBox">
              <property name="text">
               <string>&amp;Estados contables</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Recordar al arrancar</string>
           </property>
           <widget class="QCheckBox" name="empresacheckBox">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>44</y>
              <width>82</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Empresa</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="usuariocheckBox">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>83</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Usuario</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Ordenar diario por:</string>
           </property>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>9</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <item>
             <widget class="QRadioButton" name="fecharadioButton">
              <property name="text">
               <string>Fecha</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="numradioButton">
              <property name="text">
               <string>Núm. asiento</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Estilo numérico</string>
           </property>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>9</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <item>
             <widget class="QCheckBox" name="sindecimalescheckBox">
              <property name="text">
               <string>Sin decimales</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="comaradioButton">
              <property name="text">
               <string>9.999,99</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="puntoradioButton">
              <property name="text">
               <string>9,999.99</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Visualizadores de documentos</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Extensión</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Visualizador</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="ext1lineEdit">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="vis1lineEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="ext2lineEdit">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="vis2lineEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="ext3lineEdit">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="vis3lineEdit"/>
            </item>
            <item row="4" column="0">
             <widget class="QLineEdit" name="ext4lineEdit">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="vis4lineEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="idiomagroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Idioma</string>
           </property>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QRadioButton" name="espradioButton">
              <property name="text">
               <string>&amp;Español</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="catradioButton">
              <property name="text">
               <string>Català</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="galradioButton">
              <property name="text">
               <string>Galego</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="ingradioButton">
              <property name="text">
               <string>Inglés</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="portradioButton">
              <property name="text">
               <string>Portugués</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="euskeraradioButton">
              <property name="text">
               <string>Euskera</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="filtrargroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Filtrar diario por:</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>9</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <item>
             <widget class="QRadioButton" name="ejercicioradioButton">
              <property name="text">
               <string>Ejercicio actual</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="u30diasradioButton">
              <property name="text">
               <string>Últimos 30 días</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="u30asientosradioButton">
              <property name="text">
               <string>Últimos 30 asientos</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="personalizadogroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Filtro diario inicial personalizado</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="personalizadolineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="repetidos_checkBox">
           <property name="text">
            <string>Controlar registro documentos repetidos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="filtratexcheckBox">
           <property name="text">
            <string>Filtrar ficheros TEX antes de conversión a DVI</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Rutas</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Directorio de trabajo:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="dirtrabpushButton">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Cargar fichero documento</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="keme.qrc">
             <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="directoriolineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Visor ficheros DVI:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="visorlineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Aplicación para imprimir:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="imprimirlineEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Navegador</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="navegadorlineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Ruta documentos:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="rutadocpushButton">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Cargar fichero documento</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="keme.qrc">
             <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="rutadocslineEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Ruta carga documentos:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="cargadocpushButton">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Cargar fichero documento</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="keme.qrc">
             <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="rutacargadocslineEdit"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Ruta copia PDF</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="copiapdfpushButton">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Cargar fichero documento</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="keme.qrc">
             <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLineEdit" name="copiapdflineEdit"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Editor Latex:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLineEdit" name="editortexlineEdit"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Ruta copias seguridad</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLineEdit" name="rutaseglineEdit"/>
         </item>
         <item row="8" column="1">
          <widget class="QPushButton" name="rutasegpushButton">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Cargar fichero documento</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="keme.qrc">
             <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QLineEdit" name="rutalibroslineEdit"/>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Ruta libros contables</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QPushButton" name="rutalibrospushButton">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Cargar fichero documento</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="keme.qrc">
             <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/check2.png</normaloff>:/iconos/graf/check2.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>directoriolineEdit</tabstop>
  <tabstop>visorlineEdit</tabstop>
  <tabstop>imprimirlineEdit</tabstop>
  <tabstop>navegadorlineEdit</tabstop>
  <tabstop>rutadocslineEdit</tabstop>
  <tabstop>rutacargadocslineEdit</tabstop>
  <tabstop>copiapdflineEdit</tabstop>
  <tabstop>editortexlineEdit</tabstop>
  <tabstop>rutaseglineEdit</tabstop>
  <tabstop>rutalibroslineEdit</tabstop>
  <tabstop>automaticoscheckBox</tabstop>
  <tabstop>estadoscheckBox</tabstop>
  <tabstop>empresacheckBox</tabstop>
  <tabstop>usuariocheckBox</tabstop>
  <tabstop>fecharadioButton</tabstop>
  <tabstop>numradioButton</tabstop>
  <tabstop>sindecimalescheckBox</tabstop>
  <tabstop>comaradioButton</tabstop>
  <tabstop>puntoradioButton</tabstop>
  <tabstop>ext1lineEdit</tabstop>
  <tabstop>vis1lineEdit</tabstop>
  <tabstop>ext2lineEdit</tabstop>
  <tabstop>vis2lineEdit</tabstop>
  <tabstop>ext3lineEdit</tabstop>
  <tabstop>vis3lineEdit</tabstop>
  <tabstop>ext4lineEdit</tabstop>
  <tabstop>vis4lineEdit</tabstop>
  <tabstop>espradioButton</tabstop>
  <tabstop>catradioButton</tabstop>
  <tabstop>galradioButton</tabstop>
  <tabstop>ingradioButton</tabstop>
  <tabstop>portradioButton</tabstop>
  <tabstop>euskeraradioButton</tabstop>
  <tabstop>filtrargroupBox</tabstop>
  <tabstop>ejercicioradioButton</tabstop>
  <tabstop>u30diasradioButton</tabstop>
  <tabstop>u30asientosradioButton</tabstop>
  <tabstop>personalizadogroupBox</tabstop>
  <tabstop>personalizadolineEdit</tabstop>
  <tabstop>repetidos_checkBox</tabstop>
  <tabstop>filtratexcheckBox</tabstop>
  <tabstop>dirtrabpushButton</tabstop>
  <tabstop>rutadocpushButton</tabstop>
  <tabstop>cargadocpushButton</tabstop>
  <tabstop>copiapdfpushButton</tabstop>
  <tabstop>rutasegpushButton</tabstop>
  <tabstop>rutalibrospushButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>preferencias</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>684</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
